### PR TITLE
Update product-updates.md

### PR DESCRIPTION
**Pull request by graceliching**

### CODE REVIEW

File: product-updates.md [hunk 1]
```diff
@@ -9,6 +9,63 @@ date: 2018-06-05 21:45:00 Z
...
+  Apart from support for Salesforce bulk operations, Workato has included support for [Salesforce batch operations](https://docs.workato.com/connectors/salesforce/batch-operations.html), which make use of Salesforce’s Collections API.
+
+  These actions allow the creation and update of up to 2,000 records at a time. Users should make use of these operations when they are creating or updating small batches of Salesforce records, or if they wish to perform multiple operations without using Bulk API.
+
+
+- **ServiceNow: New trigger and action**
+
+  **Search records using query:** New Search records using query action allows the use of ServiceNow queries to filter the records you want. Supports up to 50,000 records retrieved per batch.
+
+  **Scheduled record search trigger:** New Scheduled record search trigger allows execution of query on a specified schedule.Query results will be returned in batches of records. Also supports up to 50,000 records retrieved per batch.
```
Comment (zweedao) [inline product-updates.md]: missing space:

New Scheduled record search trigger allows execution of query on a specified **schedule. Query** results will be returned in batches of records. Also supports up to 50,000 records retrieved per batch.

File: product-updates.md [hunk 2]
```diff
@@ -9,6 +9,63 @@ date: 2018-06-05 21:45:00 Z
...
+
+  **Increased search action limit:** The actions **Search records, Search records (User)** and **Search records using query** can now be configured to return up to 50,000 records (up from 100)
```
Comment (zweedao) [inline product-updates.md]: missing full stop

File: product-updates.md [hunk 3]
```diff
@@ -9,6 +9,63 @@ date: 2018-06-05 21:45:00 Z
...
+
+- **Okta: Scheduled trigger**
+
+  **What's new:** The new Okta Scheduled trigger allows a scheduled search instead of real-time exit
```
Comment (zweedao) [inline product-updates.md]: missing full stop

**Comment by graceliching** [comment]
@zweedao thanks, I've addressed them

**Pull request merged**